domains/ranjith.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "gh0st-cipher",

```

your github username
```suggestion
    "username": "Mohamed-faaris",
```